torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1554,6 +1554,15 @@ int listAdd(Stack& stack) {
   push(stack, std::move(ret));
   return 0;
 }
+template <class T>
+int listInplaceAdd(Stack& stack) {
+  c10::ListPtr<T> a = c10::make_list<T>();
```

Comment:
`ListPtr` got renamed to `List` recently. You'll probably have to change that.

Also, this creates an empty list just to replace it with the list from the stack two lines below. You can avoid the extra allocation by doing:

    List<T> a = pop(stack).to<List<T>>();
    List<T> b = pop(stack).to<List<T>>();